fundingmanager_test.go

```diff
@@ -558,6 +558,28 @@ func assertNumPendingChannels(t *testing.T, node *testNode, expectedNum int) {
 		expectedNum, numPendingChans)
 }
 
+func assertNumPendingChannelsRemains(t *testing.T, node *testNode, expectedNum int) {
+	var numPendingChans int
+	for i := 0; i < testPollNumTries; i++ {
+		// If this is not the first try, sleep before retrying.
+		if i > 0 {
+			time.Sleep(testPollSleepMs * time.Millisecond)
```

Comment:
This will consume `10 * 500 ms = 5 sec` each time it is called. I think it should probably be reduced to `1 sec` ish.